VERSIONS.md

```diff
@@ -1,6 +1,6 @@
 | Version | iOS version | Android version | Common files version |
 |---------|-------------|-----------------|----------------------|
-| 5.0.0   | 4.9.0       | 5.2.1           | 3.2.4                |
+| 5.0.0   | 4.9.0       | 5.3.0           | 3.3.0                |
```

Comment:
Might want to do what I did- expand the column and make it `5.0.0-rc.1`, just for consistency with the other hybrids

Reply:
Oh derp of course

Reply:
BTW I called this "beta 1" instead of "RC 1" before I saw you used RC for Cordova. In this case I think beta might be a more appropriate name though while we do more testing.

Reply:
I was going to make an Xcode joke, but you're right. Even Apple puts `beta` on theirs.